enricher/kev/kev.go

```diff
@@ -0,0 +1,275 @@
+// Package kev provides a CISA Known Exploited Vulnerabilities enricher.
+package kev
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"slices"
+	"strings"
+
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/enricher"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/claircore/pkg/tmp"
+)
+
+var (
+	_ driver.Enricher          = (*Enricher)(nil)
+	_ driver.EnrichmentUpdater = (*Enricher)(nil)
+
+	defaultFeed *url.URL
+)
+
+const (
+	// Type is the type of data returned from the Enricher's Enrich method.
+	Type = `message/vnd.clair.map.vulnerability; enricher=clair.kev schema=https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities_schema.json`
```

Comment:
Is this the schema that actually gets returned? It seems from the comments that the schema is something like `claircore.enricher.kev.Entry`?